python-frontend/src/main/java/org/sonar/python/api/PythonPunctuator.java

```diff
@@ -83,8 +83,8 @@ public enum PythonPunctuator implements TokenType {
   LEFT_ASSIGN("<<="),
   MUL_MUL_ASSIGN("**="),
   MATRIX_MULT_ASSIGN("@="),
-  WALRUS_OPERATOR(":=");
-
+  WALRUS_OPERATOR(":="),
+  BACK_SLASH("\\")
```

Comment:
I'm a bit surprised we need to declare the backslash as a Python punctuator.
This line continuation logic already exists for regular Python, I feel we should be able to make it work for line magic as well somehow, without declaring new tokens or punctuators.

Reply:
done